src/api/tokenList/TokenListApi.ts

```diff
@@ -59,6 +59,20 @@ export class TokenListApiImpl extends GenericSubscriptions<TokenDetails[]> imple
     return this._tokensByNetwork[networkId] || []
   }
 
+
+  private mergeTokenLists(baseList: TokenDetails[], newList: TokenDetails[]): TokenDetails[] {
```

Comment:
This could be a `static` method. Not that it need to be, but maybe should. Does it have to? No. Ought to? Eh....
And other modal verbs :expressionless:

Reply:
ok ok, I know you want static. I'll give you `static`.